File: Modelo/Lander.java
package Modelo;

public class Lander {

	private Integer id=0;					// PK en base de datos
	// Artibutos del Lander
	private String nombre;
	private double impulso=0;                  // Impulso retrocohetes en m·s-2
    private int nivel_impulso=0;               // Escala discreta de niveles de impulso
    private double fuel_a_quemar;              // Fuel que se gasta en cada igninción
    private double fuel_deposito;              // Fuel que lleva el lander
    private int tiempo = 0;                    // Tiempo de simulación
    private double res_tren;				   // Resistencia del tren de aterrizaje
    private PerfilPot perfPot;				   // Nivel de potencia configurado

    

    // Constantes
    
    private final double VEL_MAX = 20.0;       // Resistencia del tren de aterrrizaje m.s-1
    private final double thrust_level[] =      // Impulso de los motores ( 10 niveles ) 
                { 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0,  8.50, 20.0};
	
    public Lander() {}
    public Lander(String nombre, double fuel_deposito, double res_tren) {
		super();
		this.nombre = nombre;
		this.fuel_deposito = fuel_deposito;
		this.res_tren = res_tren;
		this.perfPot = null;
	}
	
    public String getNombre() {		return nombre; }
	public void setNombre(String nombre) {	this.nombre = nombre;	}
	public double getImpulso() {return impulso;	}
	public void setImpulso(double impulso) {this.impulso = impulso;	}
	public int getNivel_impulso() {	return nivel_impulso;	}
	public void setNivel_impulso(int nivel_impulso) {	this.nivel_impulso = nivel_impulso;	}
	public double getFuel_a_quemar() {return fuel_a_quemar;	}
	public void setFuel_a_quemar(double fuel_a_quemar) {this.fuel_a_quemar = fuel_a_quemar;	}
	public double getFuel_deposito() {	return fuel_deposito;	}
	public void setFuel_deposito(double fuel_deposito) {this.fuel_deposito = fuel_deposito;	}
	public int getTiempo() {return tiempo;	}
	public void setTiempo(int tiempo) {	this.tiempo = tiempo;	}
	public double getPerfPot(int nivel) { 	return perfPot.potencia[nivel];   }
    public void setPerfPot(PerfilPot p) {  	this.perfPot = p;   }
	public Integer getId() 								{		return id;	}
	public void setId(Integer id) 						{		this.id = id;	}
	public double getRes_tren() {	return res_tren;	}
    
    public String toString() {
    	return nombre + "  (Fuel)  "+ fuel_deposito+ "  (Tren AT)  "+ res_tren;
    }
	
    // Auxilares ( por decidir si se usan)
    /*
    private double dist=0;                     // Distancia a la superficie m
    private double acel=0;                     // aceleración m·s-2
    private double vel=0;                      // velocidad en m· s-1
    private double dist_ant=0;                 // variable auxiliar para guardar el último valor de distancia
    private double vel_ant=0;                  // variable auxiliar para guardar el último valor de velocidad
    */


} // Lander

File: Modelo/Lander.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Lander</title>
</head>
<body>
    <?php

    class Lander{

/* Variables */

private $id = 0;
private $nombre;
private $impulso;
private $nivel_impulso;
private $fuel_a_quemar;
private $fuel_deposito;
private $tiempo;
private $res_tren;
private $perfPot;

/* Constructor */
public function __construct($nombre,$fuel_deposito,$res_tren){
    $this ->nombre = $nombre;
    $this ->fuel_deposito = $fuel_deposito;
    $this ->res_tren = $res_tren;
    $this ->perfPot = null;
}
/* Getters y setters */

public function getNombre(){
    return $this ->nombre;
}
public function setNombre($nombre){
    $this ->nombre = $nombre;
}

public function getImpulso(){
    return $this ->impulso;
}
public function setImpulso($impulso){
    $this ->impulso = $impulso;
}
public function getNivel_impulso(){
    return $this ->nivel_impulso;
}
public function setNivel_impulso($nivel_impulso){
    $this ->nivel_impulso = $nivel_impulso;
}
public function getFuel_a_quemar(){
    return $this ->fuel_a_quemar; 
}
public function setFuel_a_quemar($fuel_a_quemar){
    $this ->fuel_a_quemar = $fuel_a_quemar;
}
public function getFuel_deposito(){
    return $this -> fuel_deposito;
}
public function setFuel_deposito($fuel_deposito){
    $this -> fuel_deposito = $fuel_deposito;
}
public function getTiempo(){
    return $this -> tiempo;
}
public function setTiempo($tiempo){
    $this ->tiempo = $tiempo;
}
public function getPerfbot($nivel){
    return $this -> perfPot[$nivel];
}
public function setPerfbot($p){
    $this ->perfPot = $p;
}
public function getId(){
    return $this -> id;
}
public function setId($id){
    $this -> id = $id;
}
public function getres_tren(){
    return $this -> res_tren;
}
public function __tostring(){
    return $this-> nombre . " (fuel) " . $this-> fuel_deposito . " (Tren AT ) " . $this -> res_tren;
}
    }
    ?>

</body>
</html>